src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random RANDOM = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int max = 100;
```

Comment:
The variable 'max' should be a constant according to the style guide. Please rename it to uppercase and declare it as static final, for example, `MAX_VALUE`.

Reply:
The variable 'max' should be a constant according to the style guide. Please rename it to uppercase and declare it as static final, for example, MAX_VALUE.

Reply:
you get this error because checkstyle doesn't recognize it as a constant - because it lacks `static` modifier